src/strategies/python.ts

```diff
@@ -100,22 +100,38 @@ export class Python extends BaseStrategy {
       );
     }
 
-    if (!projectName) {
+    const packagePaths = new Set<string>();
+    if (
+      parsedPyProject?.tool?.poetry &&
+      parsedPyProject?.tool?.poetry.packages
+    ) {
+      parsedPyProject?.tool?.poetry.packages.forEach(pkg => {
+        // Use 'from' if it appears, else use 'include'.
+        // Skip glob 'extra/**/*.py by split
+        packagePaths.add((pkg.from || pkg.include).split('/')[0]);
```

Comment:
Could the glob be something like `extra/path/**/*.py`? For example, perhaps there are 2 packages nested under `extra`.